flight/Modules/Stabilization/stabilization.c

```diff
@@ -944,10 +943,11 @@ static void SettingsUpdatedCb(UAVObjEvent * ev)
 		// update rates on OP (~300 Hz) and CC (~475 Hz) is negligible for this
 		// calculation
 		const float fakeDt = 0.0025f;
+
 		if(settings.GyroTau < 0.0001f)
 			gyro_alpha = 0;   // not trusting this to resolve to 0
 		else
-			gyro_alpha = expf(-fakeDt  / settings.GyroTau);
+			gyro_alpha = (1 / (1 +  ((2.0f * (float)(M_PI) * dT) / settings.GyroTau) ));
```

Comment:
Yes, that is the right. But i thought defining the filter in time domain is maybe more difficult for people, than doing it in frequency domain.

Thats why i have written in this implementation it means  tau  = 1/Fcut based on this description: https://en.wikipedia.org/wiki/Low-pass_filter#Discrete-time_realization
And when there is interest in merging this, i prefer to rename the parameter (and also adapt the math) from GyroTau to GyroFcut (as already written).